MIP113/MIP113.md

```diff
@@ -521,7 +521,7 @@ List of current ADs:
 | Navigator | [Address](https://etherscan.io/address/0x11406a9CC2e37425F15f920F494A51133ac93072)<br>[Signature](https://etherscan.io/verifySig/19371) | [Contract](https://etherscan.io/address/0x636Ff7CA03D91D8C5614394b4cC78C642a5e2Ce5)<br>**ReFi AVC**<br>[Signature](https://etherscan.io/verifySig/19369) | [Contract](https://etherscan.io/address/0x6235c806b6A7752e47A9adc815Ba343DF62A4A0a)<br>**Resiliency AVC**<br>[Signature](https://etherscan.io/verifySig/21459) | - | [Link](https://forum.makerdao.com/t/navigator-aligned-delegate-communications/20993/11) |
 | Skynet | [Address](https://etherscan.io/address/0xd4d1A446cD5976a11bd32D3e815A9F85FED2F9F3)<br>[Signature](https://etherscan.io/verifySig/19702) | [Contract](https://etherscan.io/address/0x644c092a5ccafc425ebf133123299a6397ae97d9)<br>**Resiliency AVC**<br>[Signature](https://etherscan.io/verifySig/19701) | [Contract](https://etherscan.io/address/0xb6ca415fc42b3f96641d14280c3f3a0f078e50e5)<br>**ReFi AVC**<br>[Signature](https://etherscan.io/verifySig/19700) | - | [Link](https://forum.makerdao.com/t/skynet-ad-submission/21077) |
 | Cloaky | [Address](https://etherscan.io/address/0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818)<br>[Signature](https://etherscan.io/verifySig/19712) | [Contract](https://etherscan.io/address/0x475efaC48a0A18660A7A26eE6bd5FEBF466930f8)<br>**Regenerative Finance AVC**<br>[Signature](https://etherscan.io/verifySig/21627) | [Contract](https://etherscan.io/address/0xb335e8b70f95F28e79ACF58491751f83B0050888)<br>**Sovereign Finance AVC**<br>[Signature](https://etherscan.io/verifySig/19711) | - | [Link](https://forum.makerdao.com/t/cloaky-ad-recognition-submission/21082) |
-| Penguin Soldier | [Address](https://etherscan.io/address/0x41caF770eb0a9EeC0c9cC46F388ea5A76986E217)<br>[Signature](https://etherscan.io/verifySig/21884) | [Contract](https://etherscan.io/address/0x5d97F2af00C767b7333FDC2B58d10FAFF9F51024)<br>**Sovereign Finance AVC**<br>[Signature](https://etherscan.io/verifySig/21881) | [Contract](https://etherscan.io/address/0x4D8C9c2Cd8846533967b07BECFC437542196Ad6A)<br>**KISS AVC**<br>[Signature](https://etherscan.io/verifySig/25915) | - | [Link](https://forum.makerdao.com/t/penguin-soldier-ad-recognition-submission/21395) |
+| Penguin Soldier | [Address](https://etherscan.io/address/0x41caF770eb0a9EeC0c9cC46F388ea5A76986E217)<br>[Signature](https://etherscan.io/verifySig/21884) | [Contract](0x5b4870014313c808c374F8eD1AB5b78813EB9c7f)<br>**Resiliency AVC**<br>[Signature](https://etherscan.io/verifySig/25414) | [Contract](https://etherscan.io/address/0x4D8C9c2Cd8846533967b07BECFC437542196Ad6A)<br>**KISS AVC**<br>[Signature](https://etherscan.io/verifySig/25915) | - | [Link](https://forum.makerdao.com/t/penguin-soldier-ad-recognition-submission/21395) |
```

Comment:
@blimpa Please note that the contract address has been included, but it has not been hyperlinked to Etherscan as is our standard practice